TeachingRecordSystem/src/TeachingRecordSystem.Core/Dqt/Queries/CreateQualificationQuery.cs

```diff
@@ -0,0 +1,16 @@
+namespace TeachingRecordSystem.Core.Dqt.Queries;
+
+public record CreateQualificationQuery : ICrmQuery<Guid>
+{
+    public required Guid? PersonId { get; init; }
+    public required Guid? HECountryId { get; init; }
```

Comment:
Do all these properties have to be nullable?

Reply:
HECountryId is optional in the old GTWC Import file. It will create a qualification regardless:

- CountryId is populated and invalid.
- CountryId is populated and valid.
- CountryId is not populated.

I've changed ContactId to be none null-able though.